feature_34_FAVORITES_VIEW

Symphonie234 opened the pull request:
- Added another blade file to show list of favorited listings
- Added new controller and new web route

Symphonie234 committed:
added view for favorites and the functionality to set favorite

Symphonie234 committed:
Merge branch 'develop' into feature/34_FAVORITES_VIEW

Symphonie234 committed:
removed unneccessary spacing

ChronoDevs commented on app/Http/Controllers/FavoriteController.php:
```diff
@@ -0,0 +1,71 @@
+    public function add(Listing $listing)
+    {
+        // Create a new favorite record for the current user and listing
+        Favorite::create([
```
make error response if creation is failed

ChronoDevs commented on app/Http/Controllers/FavoriteController.php:
```diff
@@ -0,0 +1,71 @@
+            ->first();
+
+        if ($favorite) {
+            $favorite->delete();
```
make error response if deletion is failed

ChronoDevs commented on app/Http/Controllers/FavoriteController.php:
```diff
@@ -0,0 +1,71 @@
+        return back()->with('success', 'Listing removed from favorites!');
+    }
+
+    public function addToFavorites(Listing $listing)
```
add comment method above

Symphonie234 committed:
attended to the changes needed to be done in the PR

Symphonie234 commented:
- Added error responses on both _**add**_ and _**remove**_ methods in the _**FavoriteController**_.
- Added comment above the _**addToFavorites**_ method in the _**FavoriteController**_.
